app/views/surveyor/responses/form/_response_question.html.haml

```diff
@@ -36,7 +36,7 @@
     .form-group
       %p.question-description
         = raw(question.description)
-  .question-options
+  .question-options{ id: ['new', 'edit', 'preview'].include?(action_name) ? '' : 'disable-likert' }
```

Comment:
This will give us multiple elements with the same ID which breaks DOM conventions. Can we use a class instead?

Reply:
Agreed, this can cause lots of problems, especially in forms.